libs/server/Resp/PurgeBPCommand.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using Garnet.common;
+using Microsoft.Extensions.Logging;
+
+namespace Garnet.server
+{
+    public enum ManagerType : byte
+    {
+        /// <summary>
+        /// MigrationManager Buffer Pool
+        /// </summary>
+        MM,
+        /// <summary>
```

Comment:
Why not Migration, Replication, ServerSocket?

Reply:
It should not use the word socket, as that implies TCP. the same network stack can be used with RDMA etc. Maybe ServerListener

Reply:
I updated this here https://github.com/microsoft/garnet/pull/623/commits/8ce18094529c8323dd6c1eb4a14cebbb3723535f